src/containers/StructurePage/folderComponents/menuOptions/ConstantMetaField.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import CustomFieldComponent from './CustomFieldComponent';
+
+interface Props {
+  key: string;
+  initialVal?: string;
+  updateFields: (newFields: Record<string, string>) => void;
+}
+
+const ConstantMetaField = ({ key, initialVal, updateFields }: Props) => {
+  return (
+    <CustomFieldComponent
```

Comment:
Du mangler en beskrivende tekst som forteller kva dette feltet er. Feks 'Gammel id for videresending' eller nokke sånt.

Reply:
La inn en placeholder.